jwst/background/background_sub.py

```diff
@@ -57,14 +164,17 @@ def background_sub(input_model, bkg_list, sigma, maxiters):
     return bkg_model, result
 
 
-def average_background(bkg_list, sigma, maxiters):
+def average_background(input_model, bkg_list, sigma, maxiters):
     """
     Average multiple background exposures into a combined data model.
     Processes backgrounds from various DataModel types, including those
     having 2D (rate) or 3D (rateints) backgrounds.
 
     Parameters:
     -----------
+    input_model: JWST data model
```

Comment:
Not the fault of this PR, but all of the param names in the list here should have whitespace on *both* sides of the colon, i.e. "input_model : JWST data model", to keep with doc string conventions.

Reply:
Corrected throughout background_sub.py, but might still be lurking elsewhere in the other step files